.evergreen/generate_evergreen_tasks.js

```diff
@@ -98,7 +111,7 @@ function makeTask({ mongoVersion, topology }) {
 
 MONGODB_VERSIONS.forEach(mongoVersion => {
   TOPOLOGIES.forEach(topology =>
-    TASKS.push(makeTask({ mongoVersion, topology }))
+    TASKS.push(makeTask({ mongoVersion, topology, tags: ['full-suite'] }))
```

Comment:
As discussed in the team channel, `full-suite` as a tag isn't doing anything for us in evergreen, but is rather used to selectively disable tasks that are known not to run on windows. No need to add a couple hundred lines of tags to our `evergreen.yml` for a tag which is not actually used

Reply:
fixed